services/controlplane.go

```diff
@@ -81,3 +65,31 @@ func RemoveControlPlane(ctx context.Context, controlHosts []*hosts.Host, force b
 	log.Infof(ctx, "[%s] Successfully tore down Controller Plane..", ControlRole)
 	return nil
 }
+
+func deployControlHost(ctx context.Context, host *hosts.Host, etcdHosts []*hosts.Host, controlServices v3.RKEConfigServices, sidekickImage, authorizationMode string, localConnDialerFactory hosts.DialerFactory, prsMap map[string]v3.PrivateRegistry) error {
```

Comment:
just for consistency can you change this to doDeploy.. to be the same as the function in workerplane.go